src/Microsoft.ML.Data/Transforms/TransformBase.cs

```diff
@@ -254,7 +254,6 @@ public override bool IsColumnActive(DataViewSchema.Column column)
     /// Base class for transforms that operate row by row with each destination column using one
     /// source column. It provides an extension mechanism to allow a destination column to depend
     /// on multiple input columns.
-    /// This class provides the implementation of ISchema and IRowCursor.
```

Comment:
Since OneToOneTransformBase currently doesn't inherit from DataViewSchema or DataViewRowCursor, I guess it's safe to remove this.

Reply:
This comment is intended to explain that classes deriving from `OneToOneTransformBase` do not need to worry about creating a schema or a cursor, since everything necessary is already implemented in `OneToOneTransformBase`. This is still true even when we don't have the `ISchema` and `IRowCursor` interfaces:

* It has the class deriving from `DataViewRowCursor` that is used by all classes deriving from `OneToOneTransformBase`. 
* It's true that it doesn't implement `DataViewSchema` (since there is only one schema class, it is not abstract), but it does have a `Bindings` class, that has an `AsSchema` property.

---
In reply to: [368201579](https://github.com/dotnet/machinelearning/pull/4673#discussion_r368201579) [](ancestors = 368201579)

Reply:
I see. So I've added some more lines to this comment to explain what you've just said in here. Please let me know if it's clear enough.

---
In reply to: [368283641](https://github.com/dotnet/machinelearning/pull/4673#discussion_r368283641) [](ancestors = 368283641,368201579)

Reply:
Thanks. You can mention that by "don't need to worry about" we mean that the abstract property `OutputSchema` and the abstract method `GetRowCursorCore` of `TransformBase` are implemented in this class. Other than that looks clear enough.

---
In reply to: [369197668](https://github.com/dotnet/machinelearning/pull/4673#discussion_r369197668) [](ancestors = 369197668,368283641,368201579)